src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<ul><li>"can be made :" --&gt; "can be made:" </li><li>"an MTOW" --&gt; "a MTOW"</li><li>Why not continue to use <strong>bold</strong> for AR, OWE... ?</li><li>"multi-disciplinary" --&gt; "multidisciplinary"</li><li>"will be seen as a design variable" --&gt; "will be considered"?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<blockquote>the MTOW is both an output of the process and an input of one of its model. This is called a sizing loop</blockquote><p>Being an output and an input of the process is not enough to have a loop. There is also a notion of "feedback".</p><p>Besides, IMHO, this definition stands for a "loop", that is not necessarily a "sizing loop". Isn't the "sizing" term linked to the nature of variables that are solved in the loop ? And do we need to explain that ? Anyway, at the very least, I think that "this is called a sizing loop" could be replaced by "we will call it a sizing loop".</p>

Reply:
<blockquote>we can guess there will be an "ideal" AR for which the aircraft will be able to fly its design mission with the lowest MTOW</blockquote><p>We generally look for the lowest fuel consumption (MTOW is somehow correlated, but both minima do not necessarily occur with the same inputs).</p><p>Makes me think that having MTOW as an ouput of performances is a bit weird. I understand the need to keep the process simple enough, but I'm wondering if having another box would not make things clearer.</p>

Reply:
I'm realizing that I'm not consistent in this first paragraph on how I present the data of the problem sometimes it's full name (abbreviation), sometimes its abbreviation (full name). So small comment to make a note to myself to change that

Reply:
> * "can be made :" --> "can be made:"
> 
>     * "an MTOW" --> "a MTOW"
> 
>     * Why not continue to use **bold** for AR, OWE... ?
> 
>     * "multi-disciplinary" --> "multidisciplinary"
> 
>     * "will be seen as a design variable" --> "will be considered"?
> 
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/)_

Corrected in dfacc3b9

Reply:
> > the MTOW is both an output of the process and an input of one of its model. This is called a sizing loop
> 
> Being an output and an input of the process is not enough to have a loop. There is also a notion of "feedback".
> 
> Besides, IMHO, this definition stands for a "loop", that is not necessarily a "sizing loop". Isn't the "sizing" term linked to the nature of variables that are solved in the loop ? And do we need to explain that ? Anyway, at the very least, I think that "this is called a sizing loop" could be replaced by "we will call it a sizing loop".

Corrected in dfacc3b9

Reply:
> Makes me think that having MTOW as an ouput of performances is a bit weird. I understand the need to keep the process simple enough, but I'm wondering if having another box would not make things clearer.

MTOW has now its own module level function to separate it more clearly from the rest of the computation and the performances now output only the fuel computation. Corrected in 10f8e2f2 and 6d1420fb